### ISSUE TITLE
Google dropped support for InAppBrowser OAuth. Now What?

### ISSUE THREAD

**Issue by JohnBain** [sahat/satellizer#1017]
As you may know, Google [dropped support for OAuth flow within InAppBrowser](https://developers.googleblog.com/2016/08/modernizing-oauth-interactions-in-native-apps.html) for security reasons. 

Is there any way now to use Satellizer's Google OAuth login flow in an Ionic app? i.e., by forcing a popup of the phone's browser?

**Comment by tennisgent** [comment]
Yeah, I hadn't heard Google had prohibited this but after reading [that blog post](https://developers.googleblog.com/2016/08/modernizing-oauth-interactions-in-native-apps.html), this basically makes  Satellizer useless for Google authentication...

Any ideas on how to get around this? According the article, by April 20, 2017 users will no longer be allowed to authenticate via embedded web views. I can't really think of any way around it other than to use native libraries for each OAuth provider (Twitter, Google, Facebook, etc)...
